src/main/java/frc/robot/RobotContainer.java

```diff
@@ -196,12 +197,8 @@ private void configureButtonBindings() {
         controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)));
     
     controlBindings.babyBomber().ifPresent(trigger -> trigger.whileTrue(
-      new BabyBirdCommand(
-        turretSubsystem, shooterSubsystem).deadlineWith(new LEDBlinkCommand(ledSubsystem, FLASH_COLOR, 0.1))
-      .andThen(new ShootTeleopCommand(
-        drivetrain, shooterSubsystem, turretSubsystem, ledSubsystem, controlBindings.translationX(),
-        controlBindings.translationY(), STOCKPILE_MID_RED, STOCKPILE_MID_BLUE, STOCKPILE_INTERPOLATOR, 0.3)
-        .deadlineWith(new LEDAlternateCommand(ledSubsystem, Color.kBlue, Color.kOrange, Seconds.of(0.1))))));
+      autoCommands.babyBird().deadlineWith(new LEDBlinkCommand(ledSubsystem, FLASH_COLOR, 0.5)).andThen(autoCommands.shootMid()
```

Comment:
You have the baby bird `LEDBlinkCommand` in `autoCommands.babyBird()` and here, that won't work. I think you should only have it in `autoCommands.babyBird()`.